README.md

```diff
@@ -23,6 +23,15 @@ Make the following changes to `package.json`:
 }
 ```
 
+To use the Babel polyfill, add the following to the `"jest"` configuration in `package.json`:
+
+```json
+"jest": {
+  "setupEnvScriptFile": "<rootDir>/node_modules/babel-jest/polyfill.js",
+  "unmockedModulePathPatterns": ["node_modules/babel-core"]
```

Comment:
I went with `node_modules/babel-core` instead of `<rootDir>/node_modules/babel-jest` because for some reason, jest seemed to be using the babel-core from another package in my project (babelify) instead of the one in babel-jest/node_modules.

Wild guess is that Jest caches the locations of modules it's seen before? But I figure this is fine since I can't imagine anyone would want babel-core mocked anywhere.